tests/integration/individual_response/test_individual_response.py

```diff
@@ -433,6 +433,114 @@ def test_previous_from_confirm_returns_via_hub_route(self):
         self.assertInUrl(f"/individual-response/{list_item_id}/how?journey=hub")
 
 
+class TestIndividualResponseTextHandler(IndividualResponseTestCase):
+    def test_display_mobile_on_confirmation_page(self):
+        # Given I navigate to the confirmation page
+        self._add_household_no_primary()
+        self.get(self.individual_section_link)
+        self.get(self.individual_response_link)
+        self.post()
+        self.post({"individual-response-how-answer": "Text message"})
+        self.post({"individual-response-enter-number-answer": "0763921456"})
```

Comment:
Is this a synthetic number and has it been screened to check it doesn't belong to anyone?

Reply:
I've just done a quick search and it doesn't look like it is. Ofcom lists appropriate ranges for dummy numbers - https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama - so I'll create one in that range and update all of the tests.

Reply:
That's a good find - not seen it before but definitely good to know